numpy/ma/tests/test_core.py

```diff
@@ -4982,6 +4982,21 @@ class Sub(type(np.ma.masked)): pass
         assert_not_equal(repr(a), 'masked')
 
 
+class TestMaskedWhereAliases(object):
+
+    # TODO: Test masked_object, masked_equal, ...
+
+    def test_masked_values(self):
+        res = masked_values(np.array([-32768.0]), np.int16(-32768))
+        assert_equal(res.mask, [True])
+
+        res = masked_values(np.inf, np.inf)
+        assert_equal(res.mask, True)
+
+        res = np.ma.masked_values(np.inf, -np.inf)
```

Comment:
All three of these tests give the opposite results in 1.13, alarmingly

Reply:
Hmm ... Is `mask_or`  working properly with `nomask`?

Reply:
What makes you say that?

Reply:
That the tests give opposite results in 1.13? Or am I missing something here.

Reply:
`inf, inf` fails because the comparison ends up being `(inf - inf) <= inf`, which returns False, the mask
`inf, -inf` fails because the comparison ends up as `(inf + inf) <= inf`, which returns `True`, the mask

This is of course backwards.

Reply:
I was more concerned with the first case, but I see it is due to `mabs(value)`.